### DISCUSSION TITLE
How to removing verbose WARNINGS(tf) and UserWarning(numpy)

### DISCUSSION THREAD

**Discussion by scott-5** [deepmodeling/deepmd-kit#1403] [Ideas]
I install deepmd-kit by conda, and so many verbose warnings I solved, but 
"disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated" (comes from migrating(tf 1.x to tf 2.x) ) and 
"UserWarning: The NumPy module was reloaded (imported a second time). This can in some cases result in small but subtle issues and is discouraged" (comes from deepmd-kit source codes),
I don't know how to remoing both two verbose warnings. Is there possible impoving codes?

**Comment by njzjz** [comment]
These warnings are raised by tensorflow and numpy, respectively. We are not able to remove them.
